contracts/internals/tokenWhitelistable.sol

```diff
@@ -37,6 +40,13 @@ contract TokenWhitelistable is ENSResolvable {
         return _tokenWhitelistNode;
     }
 
+    /// @notice Set the ENS node for the TokenWhitelist smart contract.
+    /// @param _tokenWhitelistNode_ A new ENS node for the TokenWhitelist smart contract.
+    function _setTokenWhitelistNode(bytes32 _tokenWhitelistNode_) internal {
+        _tokenWhitelistNode = _tokenWhitelistNode_;
+        emit SetTokenWhitelistNode(_tokenWhitelistNode_);
+    }
```

Comment:
Leftover - forgot to delete it - we should add it but in another PR just so that we can update the ENS names if they change - at the moment we can only set the ENS at deploy time.

Reply:
I guess we should delete it if not used.

Reply:
It's been deleted